src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -38,7 +38,7 @@ public function getConfigTreeBuilder()
             ->children()
                 ->booleanNode('toolbar')->defaultFalse()->end()
                 ->booleanNode('intercept_redirects')->defaultFalse()->end()
-                ->scalarNode('excluded_ajax_paths')->defaultValue('^/(app(_[\\w]+)?\\.php/)?_wdt')->end()
+                ->scalarNode('excluded_ajax_paths')->defaultValue('^/((?:index|app(_[\w]+)?)\.php/)?_wdt')->end()
```

Comment:
perhaps to fully clarify the regex remove `?:` as well.. (it's not really needed) since you removed escaped `\` also  (before `\\`, but also not really needed).